src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java

```diff
@@ -145,9 +138,10 @@ private void validateAndUpdateDwpHandlingOffice(SscsCaseData sscsCaseData, PreSu
 
     private void validateHearingOptions(SscsCaseData sscsCaseData, PreSubmitCallbackResponse<SscsCaseData> response) {
         HearingOptions hearingOptions = sscsCaseData.getAppeal().getHearingOptions();
-        if (hearingOptions != null && sscsCaseData.getAppeal().getHearingType() != null
+        if (hearingOptions != null
+            && sscsCaseData.getAppeal().getHearingType() != null
             && HearingType.ORAL.getValue().equals(sscsCaseData.getAppeal().getHearingType())
-            && !hearingOptions.isWantsToAttendHearing()) {
+            && Boolean.FALSE.equals(hearingOptions.isWantsToAttendHearing())) {
```

Comment:
Static import on FALSE
In fact, is isWantsToAttendHearing returns the primitive boolean, then the way this was before is better.  You may have changed it to this because it returns a Boolean object though.  Please check